### DISCUSSION TITLE
[Unstable] Europa's Finest achievement unlocks on non-ironman campaigns

### DISCUSSION THREAD

**Discussion by bananaman6213** [FakeFishGames/Barotrauma#14803] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When completing a multiplayer campaign (modded, but no mods that should have influenced this), all players got the Europa's Finest achievement at the end of the game, despite it not being a permadeath or ironman campaign.

Our host has a separate ironman campaign, though claims those settings were never enabled on this one

Attached is the save data from after the ending
[Gears gaming Scientist job.zip](https://github.com/user-attachments/files/17333362/Gears.gaming.Scientist.job.zip)
(sponsored by gears gaming (tm))

### Reproduction steps

1. Start a multiplayer campaign with ironman disabled
2. Finish the campaign and get the achievement

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Ek-Videogames** [comment]
Oh interesting, in testing our campaign never got far enough to finish as we gave up after the 3rd biome - mainly due to us thinking that the achievement's were bugged. (which they likely were at the time as this was in last unstable version, 1.6.12.0 according to the multiplayer save file) as none of the other ironman related achievements had unlocked ("The Abyss Beckons" and "Get Out Alive" respectively - for those who had died/survived)

**Comment by Regalis11** [comment]
Should be fixed now, and the fix is included in the latest unstable build.
